crates/sui-core/src/unit_tests/gas_tests.rs

```diff
@@ -120,9 +120,13 @@ async fn test_native_transfer_sufficient_gas() -> SuiResult {
     let gas_size = gas_object.object_size_for_gas_metering();
 
     gas_status.charge_storage_read(obj_size + gas_size)?;
-    gas_status.charge_storage_mutation(obj_size, obj_size, 0.into())?;
-    gas_status.charge_storage_mutation(gas_size, gas_size, 0.into())?;
-    assert_eq!(gas_cost, &gas_status.summary());
+    gas_status.charge_storage_mutation(obj_size, 0.into())?;
+    gas_status.charge_storage_mutation(gas_size, 0.into())?;
+    // computation gas is for both read and writes, so on a simple transfer is twice the objects
+    // (transfer object and gas coin)
+    gas_status.charge_computation_gas_for_storage_mutation((obj_size + gas_size) as u64 * 2)?;
```

Comment:
This makes sense to me given that the computation charging used to happen inside `charge_storage_mutation`, but now does not.